homework1/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Work 1</title>
+  <link rel="icon" type="image/x-icon" href="./img/logo.png">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+
+    <header class="header">
+
+      <div class="header--content">
+        <div class="logo"><a href="#">
+            <img src="./img/logo.png" alt="" srcset="">
+          </a></div>
+        <input class="menu-btn" type="checkbox" id="menu-btn" />
+        <label class="menu-icon" for="menu-btn"><span class="nav-icon"></span></label>
+        <ul class="menu">
+          <li><a href="#" target="_self">Course</a></li>
+          <li><a href="#" target="_self">Teacher</a></li>
+          <li><a href="#" target="_self">Price</a></li>
+          <li><a href="#" target="_self">About us</a></li>
+          <li class="menu__btn">
+            <a class="btn" href="#" target="_self">
+              Apply now
+            </a>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <div class="content">
+      <section class="content--one">
+        <div class="content--one__left">
+          <h3>
+            Desperate to learn drawing? We’ll teach you!
+          </h3>
+          <div>
+            <p>Our school has more than 20 years of experience in teaching people who always thought that they don’t
```

Comment:
try to add line break after tab.  It will be easier for code reading
```
<p>
text
</p>
```

Reply:
The text is being, but I have formatted it in a well readable format